src/test/mina_automation/archive_blocks.ml

```diff
@@ -1,18 +1,25 @@
 (**
 Module to run archive_blocks utility for the given list of block files and an archive PostgreSQL database.
 *)
-open Executor
 
 open Core
-include Executor
 
-let of_context context =
-  Executor.of_context ~context
-    ~dune_name:"src/app/archive_blocks/archive_blocks.exe"
-    ~official_name:"mina-archive-blocks"
+module Paths = struct
+  let dune_name = "src/app/archive_blocks/archive_blocks.exe"
+
+  let official_name = "mina-archive-blocks"
+end
+
+module Executor = Executor.Make (Paths)
```

Comment:
Given the previous argument that we're unwrapping when creating the module, would this boom at compile time if we feed in garbage names?

Reply:
Not fully follow. What previous argument?

Reply:
> So is this true that once we constructed a functor where standalone_path doesn't exist, we would panic immediately? That seems like what this is doing here. O.w. we should always at least guard this with a (), so it's a function.